frontend/src/metabase/dashboard/components/DashboardInfoSidebar/DashboardDetails.tsx

```diff
@@ -0,0 +1,121 @@
+import cx from "classnames";
+import { useState } from "react";
+import { c, t } from "ttag";
+
+import { skipToken, useGetUserQuery } from "metabase/api";
+import { SidesheetCardSection } from "metabase/common/components/Sidesheet";
+import DateTime from "metabase/components/DateTime";
+import Link from "metabase/core/components/Link";
+import Styles from "metabase/css/core/index.css";
+import { getUserName } from "metabase/lib/user";
+import { DashboardPublicLinkPopover } from "metabase/sharing/components/PublicLinkPopover";
+import { Box, FixedSizeIcon, Flex, Text } from "metabase/ui";
+import type { Dashboard } from "metabase-types/api";
+
+import SidebarStyles from "./DashboardInfoSidebar.module.css";
+
+export const DashboardDetails = ({ dashboard }: { dashboard: Dashboard }) => {
+  const lastEditInfo = dashboard["last-edit-info"];
+  const createdAt = dashboard.created_at;
+
+  // we don't hydrate creator user info on the dashboard object
+  const { data: creator } = useGetUserQuery(dashboard.creator_id ?? skipToken);
+
+  return (
+    <>
+      <SidesheetCardSection title={t`Creator and last editor`}>
+        {lastEditInfo && (
+          <Flex gap="sm" align="top">
+            <FixedSizeIcon name="ai" className={SidebarStyles.IconMargin} />
+            <Text>
+              {c("{0} is a date/time and {1} is a person's name").jt`${(
+                <DateTime
+                  unit="day"
+                  value={lastEditInfo.timestamp}
+                  key="date"
+                />
+              )} by ${getUserName(lastEditInfo)}`}
+            </Text>
+          </Flex>
+        )}
+
+        {creator && (
+          <Flex gap="sm" align="top">
+            <FixedSizeIcon name="pencil" className={SidebarStyles.IconMargin} />
+            <Text>
+              {c("{0} is a date/time and {1} is a person's name").jt`${(
+                <DateTime unit="day" value={createdAt} key="date" />
+              )} by ${getUserName(creator)}`}
+            </Text>
+          </Flex>
+        )}
+      </SidesheetCardSection>
+      <SidesheetCardSection title={t`Saved in`}>
```

Comment:
This is the kind of spot where a ttag contextual note would really shine, I think. It could say something like this:

"This is a heading that appears above a collection that a dashboard is saved in. Feel free to translate this as though it said 'Saved in collection', if you think that would make more sense in your language."

I'm thinking that these contextual notes will be useful, not only for POEditor translators but also professional editors and/or LLMs.